kbase-extension/static/kbase/js/common/runtime.js

```diff
@@ -32,6 +33,12 @@ define([
             };
         }
 
+        function destroy() {
+            clock.stop();
+            $(document).off('dataUpdated.Narrative');
+            theBus = null;
+            window.kbaseRuntime = null;
+        }
```

Comment:
Added a function to explicitly destroy the runtime as some parts of it kept on hanging around in some tests, despite being told in no uncertain terms that they were not welcome any more.